test/models/test_prediction.py

```diff
@@ -27,7 +27,7 @@ class PredictionTest(unittest.TestCase):
 
     def test_should_check_if_batch_frames_equivalent_to_number_of_predictions(
             self):
-        batch = Batch(frames=[Frame(1, np.ones((1, 1)), None)])
```

Comment:
This test was supposed to be updated to work with the `pd.DataFrame` change.

Reply:
Prediction Removed